### ISSUE TITLE
Images are not displaying in iOS

### ISSUE THREAD

**Issue by marklanham** [newrelic/newrelic-react-native-agent#44]
I have just installed `"newrelic-react-native-agent": "^0.0.7"` and now my iOS application is not displaying images.

I'm running:
"react-native": "0.66.5"
MacOS Monterey v12.6
Macbook Pro (14-inch, 2021, M1 Pro chipset)

**Comment by kthai-nr** [comment]
Hello, are you still having issues with #43? We're aware of an issue with images loading improperly in due to Flipper. The current solution is to comment this line in the React Native generated Podfile and running `pod install`.

```
    # Note that if you have use_frameworks! enabled, Flipper will not work and
    # you should disable the next line.

    :flipper_configuration => FlipperConfiguration.enabled,
```

Images should show after FlipperKitNetworkPlugin is disabled. Since Flipper is doing some instrumentation, it interferes with our agent and causes this problem. If you need to use Flipper, you can try to use it in debug mode and have New Relic in release mode.

**Comment by YanaSyritskaya** [comment]
Also have same problem, images are not displayed after adding this newrelic to ios.

**Comment by ndesai-newrelic** [comment]
@YanaSyritskaya  Flipper is disrupting the Objective-C runtime with [improper swizzling ](https://blog.newrelic.com/engineering/right-way-to-swizzle/). Flipper is changing the Selector names of methods in NSURLSession, but this issue can be resolved if the FlipperKitNetworkPlugin is disabled.

**Comment by YanaSyritskaya** [comment]
@ndesai-newrelic Unfortunately this way not appropriate for us. We need Flipper and cannot disable it.

**Comment by ekremtas** [comment]
if use_flipper!() is used, this bug is still going

**Comment by murilokrugner** [comment]
Also have same problem!

**Comment by ndesai-newrelic** [comment]
@murilokrugner  it will be fixed in our next release.

**Comment by kthai-nr** [comment]
Hello, @murilokrugner @ekremtas @YanaSyritskaya @marklanham  this issue should be fixed in release v1.1.0 without disabling Flipper. You can find the release here: https://www.npmjs.com/package/newrelic-react-native-agent/v/1.1.0. Please let us know if you run into any more issues.
